### ISSUE TITLE
Document how to add a filesharing mount for Windows 10

### ISSUE THREAD

**Issue by gbraad** [minishift/minishift#423]
After finishing #316, it is needed to document some of the findings related to the usernames that have to be used to mount the fileshare.

  * #422 "Account is bound to a Microsoft Account"

**Comment by thatdocslady** [comment]
@gbraad is this ready to be tackled now? It was a little confusing navigating thru the issue you lined, and also I saw that in PR #449 you added some docs already :) if the content is there, maybe I can give it a quick proofread. Also in general (cc @hferentschik ) I'm happy to see some dev tasks that involve writing docs, but I'm thinking maybe it'll be prudent to involve at least one of the writers just to give a quick pass on the language and style so that we don't need to hunt down pieces of new content (for example the uninstall doc that I edited recently). It shouldn't be too much effort, now with the new GH review request options, all you need to do is select @Preeticp or myself as reviewers. WDYT?

**Comment by gbraad** [comment]
@thatdocslady I agree with future documentation tasks being reviewed by one of you. The current write-up happened in #449. It is a technical explanation ATM, but we hope to resolve this...

@budhrg At the moment we are using External Virtual Switch for the Hyper-V setup (as a recommendation). This means that the share can be mounted as:

```powershell
PS C:\work\bin> .\minishift.exe ssh "sudo mount -t cifs //$env:computername/Users /Users -o username=gbraad@redhat.com,password=[pass],domain=$env:computername"
```
Since you were trying this earlier and had some issues, can you try this command? And also, maybe it could be helpful to mention: "Everyone" needs to be added to the "Users" share. Just wonder why you had to do this. I'll test with some Windows setups and see what is up with this.

**Comment by LalatenduMohanty** [comment]
@thatdocslady @Preeticp I think it would be a good practice to tag you folks for reviewing the doc PRs. We used to do that in ADB.

**Comment by gbraad** [comment]
part of the confusion came from a rename, will resulted in having two tasks
with a "Document" in the title. I changed the other task reflect the
technical nature. And as mentioned, it is likely a good idea to tag you,
but also include a status label.

**Comment by budhrg** [comment]
@gbraad 
> PS C:\work\bin> .\minishift.exe ssh "sudo mount -t cifs //$env:computername/Users /Users -o username=gbraad@redhat.com,password=[pass],domain=$env:computername"

This command seems to be not working for me. Getting following error:
```
mount error: could not resolve address for DESKTOP-8MV1CPM: Unkown error
```

> Since you were trying this earlier and had some issues, can you try this command? And also, maybe it could be helpful to mention: "Everyone" needs to be added to the "Users" share. Just wonder why you had to do this. I'll test with some Windows setups and see what is up with this.

I don't know but looks like I need to do that first, now when I remove that still the mount is working.

**Comment by gbraad** [comment]
@budhrg on VirtualBox? Expected, as vbox does not set a DNS that can resolve local addresses to an IP address: DESKTOP-8MV1CPM.

**Comment by hferentschik** [comment]
Merged via pull request #478.
